### DISCUSSION TITLE
Is it possible to customize Google OAuth screen URL?

### DISCUSSION THREAD

**Discussion by paulshen** [supabase/supabase#977] [Questions] [answered]
When integrating Google provider in Supabase auth, this is what the Google login screen looks like for me (notice the **continue to supabase.io**)

![image](https://user-images.githubusercontent.com/2266187/112889194-2d032700-908a-11eb-9ac5-e476395db46d.png)

Is there a way to customize it so it doesn't say "supabase.io"? Maybe host a redirect and customize the supabase auth JS library to set that redirect as the URL?

**Comment by jiholim** [comment, marked as answer]
Hi @kiwicopple @inian, is it possible to customise the url?

**Reply by inian** [reply]
Check out this issue @jiholim https://github.com/supabase/supabase/issues/1798
